lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -32,43 +32,31 @@ def initialize(app, options = {})
           @app = app
         end
 
-        def configure
```

Comment:
don't think we need this because the `Context` is reset in the `ensure` body, and we can grab settings directly from the `Datadog.configuration` registry.